app/models/github_repo.rb

```diff
@@ -27,9 +27,8 @@ def self.upsert(user, **params)
   end
 
   def self.update_to_latest
-    where("updated_at < ?", 26.hours.ago).ids.each do |repo_id|
-      GithubRepos::RepoSyncWorker.perform_async(repo_id)
-    end
+    ids = where("updated_at < ?", 26.hours.ago).ids.each_slice(1).to_a
```

Comment:
This is slick, but feels a bit too clever. Can we clarify it some?

```suggestion
    ids = where("updated_at < ?", 26.hours.ago).ids.map do |id|
      [id]
    end
```

It takes more characters, but I think it's clearer that it's wrapping each `id` in an inner array.

Reply:
Fair. I'm pretty sure I copy/pasted this off the Sidekiq wiki, agreed it's not the easiest to read. I didn't accept your suggestion because I realized we can also simplify the `where` to use a beginless range (`where(updated_at: ...26.hours.ago)`).